packages/mdc-select/README.md

```diff
@@ -242,113 +183,36 @@ is outlined below.
-| `getValueForOptionAtIndex(index: number) => string` | Returns the value for the option at the specified index within the select's menu. We adhere to the conventions of `HTMLSelectElement` - as described above - returning the value of the selected option's `id` in place of a `value` attribute and falling back to its `textContent`. Framework implementations may want to customize this method to suit their needs. |
-| `setAttrForOptionAtIndex(index: number, attr: string, value: string) => void` | Sets an attribute `attr` to value `value` for the option at the specified index within the select's menu. |
-| `rmAttrForOptionAtIndex(index: number, attr: string) => void` | Removes an attribute `attr` for the option at the specified index within the select's menu. |
-| `getOffsetTopForOptionAtIndex(index: number) => number` | Returns the `offsetTop` of the option element at the specified index. The index is guaranteed to be in bounds. |
-| `registerMenuInteractionHandler(type: string, handler: EventListener) => void` | Registers an event listener on the menu component's root element. Note that we will always listen for `MDCMenu:selected` for change events, and `MDCMenu:cancel` to know that we need to close the menu. If you are using a different events system, you could check the event type for either one of these strings and take the necessary steps to wire it up. |
-| `deregisterMenuInteractionHandler(type: string, handler: EventListener) => void` | Opposite of `registerMenuInteractionHandler`. |
-| `notifyChange() => void` | Broadcast a change event, similar to the `change` event emitted by an `HTMLSelectElement`. While we use custom events in our implementation for this, you can use any mechanism desired for notifications, such as callbacks, reactive streams, etc. Note that you can also pass data within your event if you'd like via `foundation.getValue()` and `foundation.getSelectedIndex()`. |
-| `getWindowInnerHeight() => number` | Returns the `innerHeight` property of the `window` element. |
-
-### The full foundation API
-
-#### MDCSelectFoundation.getValue() => string
-
-Returns the value of the currently selected option, or an empty string if no option is selected.
-
-#### MDCSelectFoundation.getSelectedIndex() => number
-
-Returns the index of the currently selected option. Returns -1 if no option is currently selected.
-
-#### MDCSelectFoundation.setSelectedIndex(selectedIndex: number) => void
-
-Sets the selected index of the component.
-
-#### MDCSelectFoundation.isDisabled() => boolean
-
-Returns whether or not the select is disabled.
-
-#### MDCSelectFoundation.setDisabled(disabled: boolean) => void
-
-Enables/disables the select.
-
-## Theming
-
-The select's bottom border is set to the current theme's primary color when focused.
+| `floatLabel(value: boolean) => void` | Floats or defloats label as necessary. |
```

Comment:
This is a simple on/off with no in-grained logic now, right? I'd remove "as necessary"